### DISCUSSION TITLE
Pressure Advance tuning feature sneak peek

### DISCUSSION THREAD

**Discussion by SoftFever** [OrcaSlicer/OrcaSlicer#16] [General]
![A4@7S~L75J(6FO)%B{2713K](https://user-images.githubusercontent.com/103989404/205692687-027dfa41-f948-409c-ab77-80003c586d25.png)

**Comment by harriettubgirl** [comment]
Sweet, can't wait to see it. Great work so far!

**Comment by Learoy1** [comment]
oh HELL YES!

**Comment by JamShady** [comment]
Firstly, great stuff adding this natively to OrcaSlicer.

Given that you are going to the effort to add this, what is the possible of also considering the updated [pattern method](https://ellis3dp.com/Print-Tuning-Guide/articles/pressure_linear_advance/pattern_method.html) which by many accounts is superior? The code is in [this repo](https://github.com/AndrewEllis93/Pressure_Linear_Advance_Tool), and you can [see it here](https://ellis3dp.com/Pressure_Linear_Advance_Tool/).

Cheers

**Reply by jeremytodd1** [reply]
This is currently being worked on and is very close to release.
They are working on some bug fixes at the moment. (several actions were submitted just hours ago currently)

Follow progress here: https://github.com/SoftFever/OrcaSlicer/pull/1547
